index.html

```diff
@@ -3447,6 +3447,19 @@ <h3>Syntactic Sugar</h3>
         </section>
 
       </section>
+
+      <section>
+        <h3>Mapping to the Standard Representation</h3>
+
+        <p>
+The standard representation of a Verifiable Credential is defined in this specification (which uses the media type <code>application/vc+ld+json</code>).
+Other representations can be used, provided that a mapping is defined between that representation and the standard one.
```

Comment:
```suggestion
Other representations created by the VCWG can be used, provided that a mapping is defined between that representation and the standard one.
```
Representations defined by other organizations will never be W3C VCs. This language currently implies that any other representation that maps to VCs can call itself a VC.

It would be trivial to construct an arbitrarily ridiculous representation, create a mapping and call that a VC.

I don't believe this language would pass muster when it comes to CR.

This is why the resolution itself is explicit about limiting the scope of the resolution:
> (defined by the VCWG) 

It is not our place to direct others about what they may call Verifiable Credentials. 

I also don't know what "Other representations can be used" means. Of course, people can use whatever representations they want for whatever they want. What does that have to do with W3C Verifiable Credentials?

It's also not appropriate to state that representations created by something other than the W3C are in, fact, W3C Verifiable Credentials.

It is also not appropriate to use the term Verifiable Credentials to mean anything OTHER than W3C Verifiable Credentials.

I've suggested this change to clarify that scoping.

Reply:
-1 to "created by the VCWG"

Broad adoption of VCs would be hamstrung if every use case required a special dispensation from the WG. This becomes impossible once the WG disbands.

Reply:
I agree it would be hard if the W3C VCWG has to create all mappings. The VCWG does not have subject matter or domain expertise to create all possible mappings.

For that reason, also -1 "created by the VCWG".

Reply:
-1 to "created by the VCWG", we passed a resolution specifically to remove this requirement.

Reply:
I will request changes on this PR, if this suggestion is accepted.

Reply:
agree with Orie, we explicitly noted that representations need not be created by the W3C

Reply:
https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-02-16-vcwg#resolution1

Reply:
```suggestion
Other representations can be used, provided that a mapping is defined for any
instances in that representation to the standard one.
```
i can live with `for any` I think

Reply:
The resolution explicitly includes:
> Transformation rules MUST be defined, but not necessarily by this WG.